cibuildwheel/platforms/windows.py

```diff
@@ -248,7 +247,7 @@ def setup_python(
     if build_frontend == "build[uv]" and not can_use_uv(python_configuration):
         build_frontend = "build"
 
-    use_uv = build_frontend == "build[uv]"
+    use_uv = build_frontend in {"build[uv]", "uv"}
```

Comment:
```suggestion
    use_uv = build_frontend == "build[uv]"
```

"uv" is not there yet, I would have expected an error from the linter saying something like "condition always false for `uv`"